### PR TITLE
Update getting started per latest UI

### PR DESCRIPTION
**Pull request by sterobin**
@pmuir and @jbyrne-redhat per our UX meeting this morning, I've updated the docs with the "wait several minutes" language and also made some other updates against the latest UI for the publish this Friday. I confirmed that all other quick starts don't seem to have impact from the UI changes.

Would you two mind reviewing and verifying the changes here and in the corresponding operator PR? Pete, feel free to ping a different SME if you prefer.

Corresponding PR: [Operator PR #286](https://github.com/redhat-developer/app-services-operator/pull/286)

### CODE REVIEW

File: getting-started/README.adoc [hunk 1]
```diff
@@ -99,13 +99,15 @@ endif::[]
 .Kafka instance configuration details
 image::sak-configure-kafka-instance.png[Image of instance configuration details in Create Kafka instance window]
 
-The new Kafka instance is listed in the instances table. When the instance *Status* is *Ready*, you can start using the Kafka instance. You can use the options icon (three vertical dots) to view instance details, connect to the instance, or delete the instance.
+The new Kafka instance is listed in the instances table. In some cases, you might need to wait several minutes for the instance creation process to complete.
```
Comment (jbyrne-redhat) [inline getting-started/README.adoc:102]: Suggest "...you might need to wait several minutes for the instance creation process to _finish_".

Per IBM style:
"Use _complete_ only as a transitive verb. For example, write "the program completes the operation successfully", not "the program completes".

File: getting-started/README.adoc [hunk 2]
```diff
@@ -99,13 +99,15 @@ endif::[]
...
+
+When the instance *Status* is *Ready*, you can start using the Kafka instance. You can use the options icon (three vertical dots) to view instance details, connect to the instance, change the instance owner, or delete the instance.
 
 [.screencapture]
 .Kafka instance options menu
 image::sak-kafka-instance-options.png[Image of Kafka instance options menu]
 
-NOTE: Although you can see Kafka instances created by other users in your organization, you can't manage or connect to those instances. Only the user who creates an instance can edit or delete the instance, access the associated service account and topics, or connect to the instance.
+NOTE: Although you can see Kafka instances created by other users in your organization, you might not be able manage or connect to those instances. Only the instance owner or users with permissions to access the instance can edit or delete the instance, access the associated service account and topics, or connect to the instance.
```
Comment (jbyrne-redhat) [inline getting-started/README.adoc:110]: Small typo here. "...you might not be able **to**..."

File: getting-started/README.adoc (hunk 1 above)
Comment (sterobin) [reply getting-started/README.adoc:102]: Great catch! Absolutely, fixed. Thank you!

**Review by jbyrne-redhat — APPROVED** [review]
Looks good, @sterobin  - just two small things I noticed.

**Comment by sterobin** [comment]
Got Pete's approval in chat. Merging.

**Pull request merged**